setup.py

```diff
@@ -39,7 +39,7 @@
 builtins._ASTROPY_PACKAGE_NAME_ = PACKAGENAME
 
 # VERSION should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
-VERSION = '1.13.dev'
```

Comment:
Why remove the dev tag?

Reply:
I think Joe wants to release this.

Reply:
I don't know how I feel about sneaking in a release tag in this PR, but I am not the one doing the release. Just as long as release going to happen immediately after merge *and* "dev" is added back after the release.